libsrc/agat/exehdr.s

```diff
@@ -0,0 +1,40 @@
+;
+; Oliver Schmidt, 2012-06-10
+;
+; This module supplies an AppleSingle version 2 file header + entry with
+; ID 11 according to https://tools.ietf.org/rfc/rfc1740.txt Appendix A.
+;
+
```

Comment:
Please fix (remove?) this comment, i don't think it still applies?

Reply:
Actually apple header is used for compatibility with apple commander.
Agat uses the same DOS 3.3 filesystem as Apple 2, so at least for 140k disks they are compatible.

Reply:
Oh, interesting (perhaps state that in the comment?)